.github/workflows/go.yml

```diff
@@ -51,12 +49,7 @@ jobs:
     - name: Run Test
       run: |
         go test -short ./...
-        foreach ($line in (Get-Content wix/pluginlist.txt) -as [string[]]) {
-            powershell -Command "go list $line"
-            if ($LASTEXITCODE -ne 0) {
-              exit 1
-            }
-        }
```

Comment:
This checks was needed for verifying import-paths listed in **pluginlist.txt** but now plugins are managed by **go.mod** so it is become unnecessary.